doc/specs/#2634 - Broadcast Input/#2634 - Broadcast Input.md

```diff
@@ -0,0 +1,353 @@
+
+![NetworkTower Segoe UI Icon](broadcast-segoe-icon.png)
+
+in the tab when a pane is being broadcasted to. If all tabs are being
+broadcasted to, then they'll all have that icon.
+
+It probably makes the most sense to have pane titlebars (#4998) also display
+that icon.
+
+In the original PR, it was suggested to use some variant of the [accent color]
+to on the borders of panes that are currently receiving broadcasted input. This
+would be a decent visual indicator that they're _not_ the active pane, but they
+are going to receive input. Something a bit like:
+
+![A sample of using the border to indicate the broadcasted-to panes](broadcast-input-borders.gif)
+
+iTerm2 also supports displaying "stripes" in the background of all the panes
+that are being broadcast too. That's certainly another way of indicating this
+feature to the user. I'm not sure how we'd layer it with the background image
+though.
+
+## Potential Issues
+
+<table>
+
+<tr>
+<td><strong>Compatibility</strong></td>
+<td>
+
+[comment]: # Will the proposed change break existing code/behaviors? If so, how, and is the breaking change "worth it"?
+
+</td>
+</tr>
+</table>
+
+[comment]: # If there are any other potential issues, make sure to include them here.
+
+### Future Considerations
+
+Let's look to iTerm2, who's supported this feature for years, for some
+inspiration of future things we should be worried about. If their users have
+asked for these features, then it's inevitable that our users will too 😉
+
+* [iterm2#6709]: Broadcast Input to multiple windows
+  - This is pretty straightforward. Would require coordination with the Monarch
+    though, and I'm worried about the perf hit of tossing every keystroke across
+    the process boundary.
+  - I suppose this would be `{ "action": "toggleBroadcastInput", "scope":
+    "openWindows" }` (or something with a less ridiculous name.)
+* [iterm2#6451], [iterm2#5563]: "Broadcast commands"
+  - iTerm2 has an action that lets the user manually clear the terminal-side
+    buffer. (This is tracked on the Windows Terminal as [#1882]). It might make
+    sense for there to be a mode where some _actions_ are also broadcast to
+    panes, not just key strokes. But which actions would those be? Moving the
+    selection anchors? Copy doesn't really make sense. Paste _does_ though.
+    Maybe the open find dialog / next&prev search match actions?
+  - This probably would require it's own spec.
+* [iterm2#6007]: Different stripe color for different broadcast modes
+  - Have one color to indicate when broadcasting in `global` scope, another in
+    `tab` scope, a third in `pane` scope.
+  - This might mesh well with theming ([#3327]), for properties like
+    `pane.broadcastBorderColor.globalScope`,
+    `pane.broadcastBorderColor.paneScope`. Don't love those names, but you get
+    the idea.
+* **[iterm2#5639]: Broadcast groups**, [iterm2#3372]: Broadcast Input to
+  multiple but not all tabs

```

Comment:
Wouldn't you just be able to toggle broadcast input for a tab? And using proposal 2, it's added to the set?

I was thinking you could even add this as an option in the tab's context menu. And that just adds the icon to the tab.